ARM.CMSIS.pdsc

```diff
@@ -681,21 +679,21 @@
     </component>
 
     <!-- OS Tick -->
-    <component Cclass="CMSIS" Cgroup="OS Tick" Csub="SysTick" Capiversion="1.0.1" Cversion="1.0.0" condition="OS Tick SysTick">
```

Comment:
This change must be kept.

Reply:
Cversion increased to 1.0.5.
Please see Rob's comment.